### ISSUE TITLE
Add "npm test"

### ISSUE THREAD

**Issue by Restuta** [JuniLearning/juni-eslint-config#4]
As part of this setup, it would be extremely helpful to debug/experiment how rules impact coding style.

It would be nice to include "tests" directory and add their example files that `npm test` will run listing on. This way we can assess what impact on the code-base new listing rules will have before publishing it to NPM and affecting all other projects.

**Comment by tristan-kenneth-tarpley** [comment]
Agreed. I meant to ask you about this. I saw that you did this from a shell script at Divvy, but when I've tried to run it I get a message saying that it's not authorized. Maybe we can walk through that. Could we just put the shell command in the package.json though?

Regardless, I'll start working on a test folder.

**Comment by Restuta** [comment]
yeah we can put shell command in package.json @tristan-kenneth-tarpley, but we also should be able to just do something like 

package.json:
```json
"test": "npx eslint ./tests"
```
